src/Bridge/Laravel/Console/Commands/GenerateDocumentationCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\ApiDocumenter\Bridge\Laravel\Console\Commands;
+
+use Illuminate\Console\Command;
+use LoyaltyCorp\ApiDocumenter\Documentation\Interfaces\GeneratorInterface;
+use RuntimeException;
+
+final class GenerateDocumentationCommand extends Command
+{
+    /**
+     * @var \LoyaltyCorp\ApiDocumenter\Documentation\Interfaces\GeneratorInterface
+     */
+    private $generator;
+
+    /**
+     * Constructor.
+     *
+     * @param \LoyaltyCorp\ApiDocumenter\Documentation\Interfaces\GeneratorInterface $generator
+     */
+    public function __construct(GeneratorInterface $generator)
+    {
+        parent::__construct();
+
+        $this->description = 'Generates application documentation.';
+        $this->signature = 'loyaltycorp:documentation:generate 
+            {name : The applications name}
+            {version : The version of the documentation}';
+
+        $this->generator = $generator;
+    }
+
+    /**
+     * Handles.
+     *
+     * @return void
+     *
+     * @throws \cebe\openapi\exceptions\TypeErrorException
+     */
+    public function handle(): void
+    {
+        $name = $this->argument('name');
+        if (\is_string($name) === false) {
+            // @codeCoverageIgnoreStart
+            // Laravel's awesome command function typehints..
+            throw new RuntimeException('name must be a string');
+            // @codeCoverageIgnoreEnd
+        }
+
+        $version = $this->argument('version');
+        if (\is_string($version) === false) {
+            // @codeCoverageIgnoreStart
+            // Laravel's awesome command function typehints..
+            throw new RuntimeException('version must be a string');
```

Comment:
Should name and version be capital or not because then they won't match the name of the parameter 🤔, error messages are hard

Reply:
These messages should only happen if laravel gives us something we're not expecting. I didnt spend all that much time on the ergonomics of this command since it isnt really doing anything (hence the use of \RuntimeException).

Do you think we should add more time into making this command nicer? I think the error messages are acceptable, but i can add a "The option (version) must be a string"?

Reply:
Looking into this further I think symfony already throws if the requested argument/option doesn't exist: https://github.com/symfony/console/blob/master/Input/InputDefinition.php#L132

So you can just cast (or handle array) like we do in subscriptions: https://github.com/loyaltycorp/subscriptions/blob/development/app/Console/Command.php#L50